apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.js

```diff
@@ -213,7 +227,7 @@ function ContactNewEdit({ id, data, close }) {
 						setCustomFieldsError={setCustomFieldsError}
 					/>
 				)}
-				{ContactManager && <ContactManager value={username} handler={handleUsername} />}
+				{ContactManager && <ContactManager value={userId} handler={handleUsername} />}
```

Comment:
the handler is for userName, but the value passed is userID? Why to change the file where userId is used?

Reply:
This part was updated, check you please check this again?